cmd/geth/main.go

```diff
@@ -132,6 +132,7 @@ var (
 		utils.PruneAncientDataFlag,
 		utils.ListenPortFlag,
 		utils.MaxPeersFlag,
+		utils.MaxPeersPerIpFlag,
```

Comment:
IP is a proper noun, so this var can be renamed to `MaxPeersPerIPFlag`. Reference: [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
accept, thank you for your input.

Reply:
The IP used below should also be capitalized.

Reply:
yes, updated, thx